### DISCUSSION TITLE
Can’t discover an outdoor thermometer

### DISCUSSION THREAD

**Discussion by lionep** [futomi/node-switchbot#203] [General]
I’m a total beginner with node switchbot, but I tried the implementation on two devices (radxa rock pi S and intel nuc) and I can’t discover the meter, it just hang on the promise switchbot.discover()

is there any pairing to do before discovering? Should the meter be in blink mode?
I saw that the device is only supported since 1.9.0, did anyone tried it?

thanks for any help

**Comment by donavanbecker** [comment]
Have you tried other devices? To see if those work?

**Reply by lionep** [reply]
you mean other switchbot devices ? unfortunately, I don't have any other kind (I have 4 outdoor thermometers).
